app/src/androidTest/java/com/android/voyageur/ui/search/SearchUserProfile.kt

```diff
@@ -0,0 +1,104 @@
+package com.android.voyageur.ui.search
+
```

Comment:
Here the file should be named SearchUserProfileTest, not just SearchUserProfile

Reply:
Great catch! Thank you!